dom-event-delegation/main.js

```diff
@@ -0,0 +1,16 @@
+const $taskList = document.querySelector('.task-list');
+
+function target(event) {
+  console.log('Target:', event.target);
+  console.log('Target tagName:', event.target.tagName);
+
+  if (event.target.tagName !== 'BUTTON') {
+    return;
+  }
+
+  const closest = event.target.closest('.task-list-item');
```

Comment:
Don't forget the `$` before the variable name! You're storing a DOM element, so we should include the dollar sign

Reply:
ye$ $ir